### ISSUE TITLE
First-frame END_AUDIO for certain texts produces zero audio frames

### ISSUE THREAD

**Issue by ldub** [mudler/voxtral-tts.c#4]
I've been hacking on adding [voxtral-tts support to mlx-audio](https://github.com/ldub/mlx-audio/pull/1). Reading your codebase has been very helpful, so thank you! I did find this issue, however.

## Reproduction

```bash
./voxtral_tts \
  -d /path/to/Voxtral-4B-TTS-2603 \
  -v neutral_female \
  -o test.wav \
  "Good morning, and welcome to the daily briefing."
```

Output:
```
Generating audio
generate: no audio frames produced
Speech generation failed
```

The path to the model on my machine is `/Users/lev/.cache/huggingface/hub/models--mistralai--Voxtral-4B-TTS-2603/snapshots/e3c858a0519f292a9eeb03594e575464244ae865`, yours will likely be similar.

## Analysis

The model predicts `END_AUDIO` (semantic code 1) on the very first decode step. The first-frame semantic logits show `END_AUDIO` and code 10 are nearly tied — `END_AUDIO` wins by less than 0.01 in float32:

```
END_AUDIO (1): 9.7917
Code 10:       9.7851
Code 855:      9.3608
```

This is deterministic for this specific text + voice combination (tested 10 consecutive runs, all zero frames).

Other texts with the same voice work fine, e.g.:
```bash
./voxtral_tts -d /path/to/model -v neutral_female -o test.wav \
  "Hello world, this is a test of the Voxtral TTS model."
# -> Generated 73 audio frames (5.84 seconds)
```

## Possible fix

Mask `END_AUDIO` in the semantic logits for the first frame. Any non-empty text input should produce at least one audio frame, and the model can still terminate naturally from the second frame onward.

## Environment

- Model: `mistralai/Voxtral-4B-TTS-2603` (snapshot `e3c858a`)
- Voice: `neutral_female`
- OS: macOS (Apple Silicon)
